os/net/lwip/configs/nd/Kconfig

```diff
@@ -3,9 +3,10 @@
 # see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
 #
 
-menu "ND Support"
+menu "Neighbor Discovery (RFC 4861)"
 
 config NET_IPv6_ND
```

Comment:
There is no code for this. Why do you need this config?

Reply:
for compatibility. there are many files to do fix.  and it's not fully tested. I'll remove it later